src/libos/src/lib.rs

```diff
@@ -78,8 +82,6 @@ use crate::process::pid_t;
 
 #[macro_use]
 mod prelude;
-#[macro_use]
-mod error;
```

Comment:
You can export the errno crate to error mod.

Reply:
Done